api/src/main/java/org/openmrs/module/metrics/builder/impl/JmxReportBuilderImpl.java

```diff
@@ -0,0 +1,31 @@
+package org.openmrs.module.metrics.builder.impl;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.jmx.JmxReporter;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.module.metrics.builder.JmxReportBuilder;
+
+public class JmxReportBuilderImpl implements JmxReportBuilder {
+	
+	protected final Log log = LogFactory.getLog(this.getClass());
```

Comment:
I probably haven't flagged this too much, but I would prefer using SLF4J over Apache Commons Logging.

```java
private static final Logger log = LoggerFactory.getLogger(JmxReportBuilderImpl.class);
```

PS Loggers should almost always be `private`, `static` and `final`.

The reason for using SLF4J is that it's a generic interface that doesn't tie us to an underlying logging library, whereas Apache Commons Logging is a specific logging implementation. The other part (that makes this complex) is that OpenMRS doesn't actually use Apache Commons Logging. All our logging is done by the Log4J library. Instead, we use SLF4J's facade to allow us to act as-if we support the Apache Commons Logging API. Switching to just using SLF4J directly will thus have less overhead.